backend/game/turn_actions/test_heal.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import game.turn_actions.heal as heal
+from game.player.player import Player
+
+
+@pytest.fixture(autouse=True)
+def hurt_player():
+    hurt_player: Player = Player()
+    hurt_player.update_health_by(-3)
+    return hurt_player
+
+
+def test_heal_from_dice_out_of_tokyo(hurt_player):
+    hurt_player.leave_tokyo()
```

Comment:
Maybe I'm being pedantic, but I don't like .leave_tokyo() here, since that's a verb, an actions, to be triggered when the player is leaving tokyo.
Instead, can we do hurt_player.location == Locations.OUTSIDE? This way it's clear that you're just setting the location, not talking about actually moving locations.

Reply:
Ok I can change that